src/content/docs/network-performance-monitoring/setup-performance-monitoring/snmp-performance-monitoring.mdx

```diff
@@ -433,13 +433,13 @@ Our network monitoring container supports all major versions of SNMP (v1, v2c, a
 ## Set up SNMP data monitoring in New Relic [#setup-snmp-monitoring]
 
 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Add more data**.
-2. Scroll down until you see **Network monitoring** and click **SNMP**.
-3. Follow the steps in New Relic.
+2. Scroll down until you see **Network** and click **SNMP**.
+3. Follow the steps outlined in the New Relic UI.  The available installation methods are a Docker or Linux package manager.
```

Comment:
Can this be changed to - "The available installation methods are Docker or Linux package manager."  Removing the "a".